src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+import { Person } from '../../types';
+
+type Props = {
+  isError: boolean;
+  people: Person[];
+  isLoaded: boolean;
+  loadDate: () => void;
+};
+
+export const PeoplePage: React.FC<Props> = ({
+  isError,
+  people,
+  isLoaded,
+  loadDate,
```

Comment:
Since you use all these variables only here it's better to move all states into this component instead of passing props